classes/press/Press.inc.php

```diff
@@ -33,7 +33,7 @@ function __construct() {
 	 * param $home boolean get homepage title
 	 * @return string
 	 */
-	function getPageHeaderTitle() {
```

Comment:
@ajnyga, could you add `@deprecated` functions with the old function names that call these two functions?

Reply:
sorry, I am not sure what you mean. An example?

Reply:
Just in case themes use these functions:
```
/**
 * @deprecated Since OMP 3.2.1, use getLocalizedPageHeaderTitle instead.
 * @return string
 */
function getPageHeaderTitle() {
	return $this->getLocalizedPageHeaderTitle();
}
```
...also for the other function.

Reply:
check